systers_portal/templates/account/signup.html

```diff
@@ -0,0 +1,63 @@
+{% extends "account/base.html" %}
```

Comment:
I assume template "account/base.html" is somewhere hidden inside django-allauth package.
What we actually want is to extend our "base.html" template defined inside systers_portal project.